### PR TITLE
[new release] hxd (0.1.0)

### PR DESCRIPTION
**Pull request by dinosaure**
Hexdump in OCaml

- Project page: <a href="https://github.com/dinosaure/hxd">https://github.com/dinosaure/hxd</a>
- Documentation: <a href="https://dinosaure.github.io/hxd/">https://dinosaure.github.io/hxd/</a>

##### CHANGES:

- First release

### CODE REVIEW

**Comment by mseri** [comment]
Thanks!

File: packages/hxd/hxd.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+name:         "hxd"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/dinosaure/hxd"
+bug-reports:  "https://github.com/dinosaure/hxd/issues"
+dev-repo:     "git+https://github.com/dinosaure/hxd.git"
+doc:          "https://dinosaure.github.io/hxd/"
+license:      "MIT"
+synopsis:     "Hexdump in OCaml"
+description: """Please, help me to debug ocaml-git
```
Comment (mseri) [inline packages/hxd/hxd.0.1.0/opam]: I find this description funny :) but do you want to update it with something more descriptive before I merge, or it's fine?

Comment (dinosaure) [reply packages/hxd/hxd.0.1.0/opam]: It's fine :+1:, the synopsis is clear and then, the goal of this package is very limited (debugging only, and used only by me).

**Comment by mseri** [comment]
Thanks

**Pull request merged**